rust/codegen/Cargo.toml

```diff
@@ -3,9 +3,9 @@ name = "suborbital-macro"
 version = "0.15.1"
```

Comment:
```suggestion
version = "0.16.0"
```

Should this be bumped now?

Reply:
I would hold off until we're ready to release.